tox.ini

```diff
@@ -90,17 +90,14 @@ install_command=python -m pip --use-deprecated=legacy-resolver install {opts} {p
 deps =
     dataclasses
     PyYAML
-    regendoc>=0.6.1
+    regendoc>=0.8.1
     sphinx
 whitelist_externals =
     rm
     make
 commands =
     # don't show hypothesis plugin info in docs, see #4602
-    pip uninstall hypothesis -y
-    rm -rf /tmp/doc-exec*
-    rm -rf {envdir}/.pytest_cache
-    make regen
```

Comment:
@RonnyPfannschmidt If you removed those, I suppose the comment should go away too, no?

Reply:
done, thanks